bugbug/model.py

```diff
@@ -275,6 +285,7 @@ def print_feature_importances(
     def train(self, importance_cutoff=0.15):
         classes, self.class_names = self.get_labels()
         self.class_names = sort_class_names(self.class_names)
+        is_multilabel = False
```

Comment:
It's a bit strange to set it as false here and then overwrite it later, I'd just do:
```Python
is_multilabel = isinstance(y[0], np.ndarray)
```
at line 301.